Add maskInverted prop to MaskExtension (#7440)
visgl/deck.gl: docs/api-reference/extensions/mask-extension.md

![maskByInstance](https://raw.githubusercontent.com/visgl/deck.gl-data/master/images/docs/mask-by-instance.png)

##### `maskInverted` (Boolean, optional)

> When `maskInverted` is true the result of the masking operation is inverted. Inversion is applied when reading the mask, thus is it possible to use the same mask normally as well as using the inverse on different layers. Defaults to `false`.

Reviewer: ```suggestion
When `maskInverted` is true the result of the masking operation is inverted. Inversion is applied when reading the mask, thus it is possible to use the same mask normally on some layers and inverted on other layers. Defaults to `false`.
```
